tools/testing/test_selections.py

```diff
@@ -98,20 +105,24 @@ def calculate_shards(
     tests: List[str],
     test_file_times: Dict[str, float],
     must_serial: Optional[Callable[[str], bool]] = None,
+    sort: bool = True,
```

Comment:
why might sort be set to false?

Reply:
Prioritized tests set it to false while doing sharding since prioritized tests already have an order.  sort here refers to sorting done in the calculate test shards, which sorts by test time since that usually results in more even sharding.
Renamed variable to sort_by_time for clarity.